backend/core/src/seeder/listings-seeder/listings-seeder.service.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { ListingsSeederService } from './listings-seeder.service';
+
+describe('ListingsSeederService', () => {
+  let service: ListingsSeederService;
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      providers: [ListingsSeederService],
+    }).compile();
+
+    service = module.get<ListingsSeederService>(ListingsSeederService);
+  });
+
+  it('should be defined', () => {
+    expect(service).toBeDefined();
```

Comment:
Do we want this placeholder test here? I saw you deleted the other ones...

Reply:
Done

Reply:
Removed.